ci/pipeline.yaml

```diff
@@ -598,4 +915,565 @@ jobs:
+        params:
+          globs:
+            - olm-*.zip
+      - get: agent-operator-release-source
+        trigger: true
+        passed: [operator-olm-github-release]
+      - in_parallel:
+          steps:
+            - do:
+                - get: community-operators-repo
+                - task: community-operator-commit-changes
+                  config:
+                    platform: linux
+                    image_resource:
+                      type: registry-image
+                      source:
+                        repository: gcr.io/instana-agent-qa/agent-operator/e2e-test-base-image
+                        tag: main
+                        username: _json_key
+                        password: ((project-berlin-tests-gcp-instana-qa))
+                    inputs:
+                      - name: instana-agent-operator-release
+                      - name: community-operators-repo
+                    outputs:
+                      - name: community-operators-repo
+                    params:
+                      GH_API_TOKEN: ((instanacd-github-api-token))
+                      OWNER: k8s-operatorhub
+                      REPO: community-operators
+                    run:
+                      path: bash
+                      args:
+                        - -ce
+                        - |
+                          pushd instana-agent-operator-release
+
+                          OLM_BUNDLE_ZIP=$(ls olm*.zip)
+                          OPERATOR_RELEASE_VERSION=$(echo $OLM_BUNDLE_ZIP | sed 's/olm-\(.*\)\.zip/\1/')
+                          COMMIT_MESSAGE="operator instana-agent-operator ($OPERATOR_RELEASE_VERSION)"
+                          OLM_BUNDLE_ZIP_PATH="$(pwd)/$OLM_BUNDLE_ZIP"
+
+                          echo "Check if the PR is already open"
+                          PR_LIST_JSON=$(curl -fH "Accept: application/vnd.github+json" https://api.github.com/repos/$OWNER/$REPO/pulls)
+                          EXISTING_PR_INFO_JSON=$(echo $PR_LIST_JSON | jq ".[] | select(.title==\"$COMMIT_MESSAGE\")")
+                          if [ ! -z "$EXISTING_PR_INFO_JSON" ]; then
+                            echo "PR already exists, exiting"
+                            exit 0
+                          fi
+                          echo "PR does not exist, creating a PR"
+
+                          popd
+
+                          pushd community-operators-repo/operators/instana-agent-operator
+                          set -x
+                          git pull -r
+
+                          mkdir -p $OPERATOR_RELEASE_VERSION
+                          unzip -o $OLM_BUNDLE_ZIP_PATH -d $OPERATOR_RELEASE_VERSION
+
+                          git config --global user.name "instanacd"
+                          git config --global user.email "instanacd@instana.com"
+
+                          git add .
+                          git commit -s -m "$COMMIT_MESSAGE" --allow-empty
+
+                          popd
```

Comment:
As discussed, we might want to switch to using feature branches on the fork instead of relying on main which might need frequent rebasing and could cause conflicts, but let's do that in a separate PR, so it is fine for now.

Reply:
Yes, I added to the list of the follow-ups:

- remove agent-operator-rh-olm-release from delivery-infra -- it's not present in delivery-infra, manual deletion is sufficient
- make olm-release-pr-creation job more robust (rebasing, branching)

As the PR is already quite large, I'd keep those changes separate